### ISSUE TITLE
tests fail in python3

### ISSUE THREAD

**Issue by ofloveandhate** [bertiniteam/b2#62]
this is perhaps due to a syntactical difference between python2 and python3, but i am getting the following error, having built PyBertini using `configure PYTHON=python3`:

```
integral:python ofloveandhate$ python3 test/
__init__.py  classes/     test_all.py  tracking/    
integral:python ofloveandhate$ python3 test/test_all.py 
Traceback (most recent call last):
  File "test/test_all.py", line 29, in <module>
    import classes.test_classes
  File "/Users/ofloveandhate/math_projects/bertini2/bertiniteam/b2/python/test/classes/test_classes.py", line 42
    print mods
             ^
SyntaxError: Missing parentheses in call to 'print'
```

**Comment by cthoyt** [comment]
I made a pull request #63 to fix this!

**Comment by cthoyt** [comment]
Maybe a better solution is to use the `logging` module, which can allow for user-specified granularity of printing statements and to what location, without changing any code inside your project

**Comment by ofloveandhate** [comment]
this issue will be fixed in an upcoming release, when #63 is incorporated.  closing.
